### ISSUE TITLE
MKtlLookup should find all descs that fit for a registered device

### ISSUE THREAD

**Issue by adcxyz** [ModalityTeam/Modality-toolkit#244]
```
// currently MKtl.find only posts the deviceID as used in MKtlLookup, e.g.:
MKtl('midi_0_nanok', 'midi_0_nanokontrol');

// It should actually post both the uniqueID above, 
// and then the fitting desc file name(s):
MKtl('midi_0_qunex', "keith-mcmillen-qunexus_port1_AB");
MKtl('midi_0_qunex', "keith-mcmillen-qunexus_port1_C");
MKtl('midi_0_nanok', "korg-nanokontrol");
```

**Comment by adcxyz** [comment]
closed with 74e7e42

**Comment by adcxyz** [comment]
post for MKtl.find(\midi) now looks as below - hope this is clear enough?
```
MIDI Sources:
	MIDIEndPoint("nanoKONTROL", "SLIDER/KNOB", -616253900)
	MIDIEndPoint("IAC Driver", "IAC Bus 1", -476939297)
	MIDIEndPoint("QuNexus", "Port 1", 823877272)
	MIDIEndPoint("QuNexus", "Port 2", -1162266210)
	MIDIEndPoint("QuNexus", "Port 3", 1506407324)
MIDI Destinations:
	MIDIEndPoint("nanoKONTROL", "CTRL", 494130505)
	MIDIEndPoint("IAC Driver", "IAC Bus 1", -1608967985)
	MIDIEndPoint("QuNexus", "Port 1", -6208339)
	MIDIEndPoint("QuNexus", "Port 2", 1341268196)
	MIDIEndPoint("QuNexus", "Port 3", -1379705999)
midi_2_qunexus numInPorts: 3, numOutPorts: 3, numInDevs: 1, numOutDevs: 1

-----------------------------------------------------

// Available MIDIMKtls: 
// MKtl('myNickName', 'lookupName');  
		// *[ midi device, portname, uid]

MKtl('midi_0_nanok', 'midi_0_nanokontrol');
		// [ "nanoKONTROL", "SLIDER/KNOB", -616253900 ]
		// create from desc file:
MKtl('midi_0_nanokontrol', "korg-nanokontrol");

MKtl('midi_1_iac_d', 'midi_1_iac_driver');
		// [ "IAC Driver", "IAC Bus 1", -476939297 ]
		// no matching desc files found!


MKtl('midi_2_qunex', 'midi_2_qunexus_port_1');
		// [ "QuNexus", "Port 1", 823877272 ]
		// multiple desc files found!
		//choose one of them to create the MKtl:
MKtl('midi_2_qunexus_port_1', "keith-mcmillen-qunexus_port1_AB");
MKtl('midi_2_qunexus_port_1', "keith-mcmillen-qunexus_port1_C");
MKtl('midi_2_qunexus_port_1', "keith-mcmillen-qunexus_port1_D");
MKtl('midi_2_qunexus_port_1', "keith-mcmillen-qunexus_port2_pocketfader");

MKtl('midi_3_qunex', 'midi_3_qunexus_port_2');
		// [ "QuNexus", "Port 2", -1162266210 ]
		// multiple desc files found!
		//choose one of them to create the MKtl:
MKtl('midi_3_qunexus_port_2', "keith-mcmillen-qunexus_port2");
MKtl('midi_3_qunexus_port_2', "keith-mcmillen-qunexus_port2_joybox");

MKtl('midi_4_qunex', 'midi_4_qunexus_port_3');
		// [ "QuNexus", "Port 3", 1506407324 ]
		// create from desc file:
MKtl('midi_4_qunexus_port_3', "keith-mcmillen-qunexus_port3");


-----------------------------------------------------
-> MKtl
```
